src/internal.c

```diff
@@ -5237,7 +5269,11 @@ int Ed25519Sign(WOLFSSL* ssl, const byte* in, word32 inSz, byte* out,
         ret = ssl->ctx->Ed25519SignCb(ssl, in, inSz, out, outSz, keyBuf,
             keySz, ctx);
     }
+#if defined(HAVE_PK_CALLBACKS) && defined(WOLFSSL_MAXQ108x)
+    if ((ret == NOT_COMPILED_IN) || ssl->options.side != WOLFSSL_CLIENT_END)
```

Comment:
Same why only client?

Reply:
See above.  Will be fixed to not check for client side.

Reply:
Fixed. Keeping fallback.